sycl/test-e2e/Basic/built-ins/marray_math.cpp

```diff
@@ -118,8 +116,7 @@ int main() {
   TEST2(sycl::remquo, float, int, 3, EXPECTED(float, 1.4f, 4.2f, 5.3f),
         EXPECTED(int, 0, 0, 0), 0.0001, ma6, ma3);
   TEST3(sycl::nan, float, 3, ma7);
-  if (deviceQueue.get_device().has(sycl::aspect::fp64))
-    TEST3(sycl::nan, double, 3, ma8);
+  TEST3(sycl::nan, double, 3, ma8);
```

Comment:
```suggestion
  if (deviceQueue.get_device().has(sycl::aspect::fp64))
    TEST3(sycl::nan, double, 3, ma8);
```
This part as well.

Reply:
I have no idea how that could have happened...